app/services/google_analytics/requests/page_views_request.rb

```diff
@@ -6,27 +6,30 @@ module Requests
     class PageViewsRequest
       include Google::Apis::AnalyticsreportingV4
 
-      def build(base_paths, start_date: "7daysAgo", end_date: "today")
+      def build(base_paths, start_dates: ["7daysAgo"], end_date: "today")
```

Comment:
What about removing the defaults for `start_dates`?. It feels a bit misleading when you read the method signature. It seems that we are not using it in our codebase.